<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>

  <!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-Y9EJC8CJW6"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-Y9EJC8CJW6');
</script>

<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Sina Majidian | Bioinformatics, Computational biologist, Comparative genomics | </title>
<meta name="description" content="My research studies and where I studied. Iran, Switzerland or Netherlands.Bioinformatics, computational biologist, Comparative genomics, Phd Student or postdoc life. ">


</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Home</div>
<div class="menu-item"><a href="index.html" class="current">About&nbsp;me</a></div>
<div class="menu-item"><a href="publications.html">Publications</a></div>
<div class="menu-istem"><a href="teaching.html" class="current">Teaching</a></div>
<div class="menu-item"><a href="research.html">Research</a></div>
<div class="menu-item"><a href="news.html">News</a></div>
<div class="menu-item"><a href="blog_persian.html">وبلاگ فارسی</a></div>
<div class="menu-category">Social Media</div>
<div class="menu-item"><a href="https://www.linkedin.com/in/sina-majidian/">Linkedin</a></div>
<div class="menu-item"><a href="https://bsky.app/profile/sinamajidian.bsky.social">BlueSky</a></div>
<div class="menu-item"><a href="https://twitter.com/SinaMajidian">Twitter</a></div>
<div class="menu-category">Links</div>
<div class="menu-item"><a href="https://github.com/sinamajidian/">GitHub</a></div>
<div class="menu-item"><a href="https://scholar.google.nl/citations?user=pgzi_1QAAAAJ">GoogleScholar</a></div>
<div class="menu-item"><a href="https://orcid.org/0000-0001-5345-6982">ORCID</a></div>
</td>
<td id="layout-content">
<div id="toptitle">

<h1>Sina Majidian</h1>
</div>
<table class="imgtable"><tr><td>
<img src="photos/sina_majidian_image.jpeg" alt="Sina Majidian's image" width="256px" height="192px" />&nbsp;</td>
<td align="left"><p><a href="https://sinamajidian.github.io">Sina Majidian</a>, Ph.D.  <br />
Postdoctoral Fellow, <a href="https://langmead-lab.org/">Langmead Lab</a>  <br />
<a href="www.cs.jhu.edu">Department of Computer Science</a>, <br />
<a href="www.jhu.edu">Johns Hopkins University</a>. <br />

<br />
Email: sina.majidian a.&t. gmail <br />


[<a href="https://scholar.google.com/citations?user=pgzi_1QAAAAJ&hl">Google Scholar</a>]<br /></p>
</td></tr></table>
<p>(Rome, Italy. Apr 24, 2022)</p>




<h2>Research Interests </h2>
<ul>
  <li><p> Comparative genomics </p>
  </li>

  <li><p> Orthology and phylogeny inference at scale </p>
  </li>

  <li><p> Genetic variations and haplotype assembly</p>
  </li>

</ul>

<h2>About me</h2>
<p> Orthology and phylogeny inference at scale, along with their applications, are my main research interests. I have developed methods and written software in the field of comparative genomics, aiming to enable evolutionary analysis at the scale of the Tree of Life. I contributed to the development of Read2Tree, a fast and accurate method for inferring phylogenies from sequencing reads. I am also the first author of FastOMA, a tool for accurately identifying orthologous genes by distinguishing them from paralogs. This tool makes a significant impact in orthology prediction and addresses the challenges of growing large-scale genomic data. I was invited to present this work at Institut Pasteur, University of Zurich, ISMB, and the Biodiversity Genomics Conference in 2023. Analysing genetic variations and haplotype assembly using mathematical modeling were focus of my PhD thesis. I have developed methods for estimating haplotype blocks from single nucleotide variants (SNVs) called from DNA sequencing reads. I have reviewed manuscripts for several journals including Genome Biology, Genome research, and NARGB. </p>


<h2>Experiences </h2>
  <ul>
    <li><p> (May 2024 – present) Postdoctoral Fellow,
    <a href="www.cs.jhu.edu"> Department of Computer Science</a>,
    Johns Hopkins University.</p>
    </li>

    <li><p> (May 2021 – Apr 2024) Postdoctoral Fellow,
    <a href="https://www.unil.ch/dbc/home.html"> Department of Computational Biology</a>,
    University of Lausanne, Switzerland.</p>
    </li>

    <li><p>(Nov. 2019 -Jan. 2020) Remote intern,
    <a href="https://www.hgsc.bcm.edu/"> Human Genome Sequencing Center</a>,
    Baylor College of Medicine, USA.</p>
    </li>

    <li><p>(Oct. 2018 - Oct. 2019) Guest researcher,
    <a href="https://www.wur.nl/en/research-results/chair-groups/plant-sciences/bioinformatics.htm">
    Bioinformatics group</a>,
    Department of Plant Sciences, Wageningen University & Research, The Netherlands.</p>
    </li>

</ul>


<h2>Education </h2>
<ul>
    <li><p>(2015 – 2020) Doctor of Philosophy in Electrical Engineering- Signal processing,
      <a href="http://www.iust.ac.ir/en">
      Iran University of Science & Technology</a>,</p>
    </li>

    <li><p>(2013 - 2015) Master of Science in Electrical Engineering- Signal processing,
      <a href="http://www.iust.ac.ir/en">
      Iran University of Science & Technology</a>,</p>
    </li>

    <li><p>(2008 - 2013) Bachelor of Science in Electrical Engineering,
      <a href="https://en.sbu.ac.ir/">
      Shahid Beheshti University, Iran.</a>.</p>
    </li>
</ul>



<h2>Remote research projects </h2>
<p>  I have had the privilege of supervising several undergraduate and graduate students. We can define a six-month project focused on developing tools and conducting analyses in comparative genomics. Interested candidates are invited to send their resumes to me.".</p>


</td>
</tr>
</table>
</body>
</html>
