<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
<title>Publications </title>
</head>
<body>
  <table summary="Table for page layout." id="tlayout">
  <tr valign="top">
  <td id="layout-menu">
  <div class="menu-category">Home</div>
  <div class="menu-item"><a href="index.html" class="current">About&nbsp;me</a></div>
  <div class="menu-item"><a href="publications.html">Publications</a></div>
  <div class="menu-item"><a href="teaching.html" class="current">Teaching</a></div>
  <div class="menu-item"><a href="experiences.html">Experiences</a></div>
  <div class="menu-item"><a href="blog.html">Blog</a></div>
  <div class="menu-item"><a href="blog_persian.html">وبلاگ فارسی</a></div>
  <div class="menu-category">Social Media</div>
  <div class="menu-item"><a href="https://www.linkedin.com/in/sina-majidian/">Linkedin</a></div>
  <div class="menu-item"><a href="https://twitter.com/SinaMajidian">Twitter</a></div>
  <div class="menu-category">Links</div>
  <div class="menu-item"><a href="https://github.com/sinamajidian/">Github</a></div>
  <div class="menu-item"><a href="https://scholar.google.nl/citations?user=pgzi_1QAAAAJ">GoogleScholar</a></div>
  <div class="menu-item"><a href="https://orcid.org/0000-0001-5345-6982">ORCID</a></div>
  </td>
  <td id="layout-content">
  <div id="toptitle">


<h1>Publications and Submissions </h1>
</div>

</ul>
<h3>Publications</h3>
<ul>
<li> OMAMO: orthology-based alternative model organism selection. Bioinformatics, 2022, [<a href="https://doi.org/10.1093/bioinformatics/btac163">paper</a>], [<a href="https://github.com/DessimozLab/omamo">code</a>], web service.
<br />
</li>
<li> Matrix completion with weighted constraint for haplotype estimation. Digital Signal Processing, 2021, paper, code.
<br />
</li>
<li> PhaseME: Automatic rapid assessment of phasing quality and phasing improvement. GigaScience, 2020, paper, code
<br />
</li>
<li> Hap10: reconstructing accurate and long polyploid haplotypes using linked reads. BMC Bioinformatics, 2020, paper, code.
<br />
</li>
<li> Minimum error correction-based haplotype assembly: considerations for long read data. PLOS one, 2020, paper, code.
<br />
</li>
<li> Haplotype assembly using manifold optimization and error correction mechanism. IEEE Signal Processing Letters, 2019, paper.
<br />
</li>
<li> NGS based haplotype assembly using matrix completion. PLOS one, 2019, paper, code.
<br />
</li>
</ul>

<h3>Submissions</h3>
<ul>
<li> Read2Tree: scalable and accurate phylogenetic trees from raw reads, Submitted, preprint, code.
<br />
</li>
</ul>
</td>
</tr>
</table>
</body>
</html>
