<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>

  <!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-Y9EJC8CJW6"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-Y9EJC8CJW6');
</script>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title> Sina Majidian's Publications </title>
</head>
<body>
  <table summary="Table for page layout." id="tlayout">
  <tr valign="top">
  <td id="layout-menu">
  <div class="menu-category">Home</div>
  <div class="menu-item"><a href="index.html" class="current">About&nbsp;me</a></div>
  <div class="menu-item"><a href="publications.html">Publications</a></div>
  <div class="menu-item"><a href="teaching.html" class="current">Teaching</a></div>
  <div class="menu-item"><a href="research.html">Research</a></div>
  <div class="menu-item"><a href="news.html">News</a></div>
  <div class="menu-category">Social Media</div>
  <div class="menu-item"><a href="https://www.linkedin.com/in/sina-majidian/">Linkedin</a></div>
  <div class="menu-item"><a href="https://bsky.app/profile/sinamajidian.bsky.social">BlueSky</a></div>
  <div class="menu-item"><a href="https://twitter.com/SinaMajidian">Twitter</a></div>
  <div class="menu-category">Links</div>
  <div class="menu-item"><a href="https://github.com/sinamajidian/">GitHub</a></div>
  <div class="menu-item"><a href="https://scholar.google.nl/citations?user=pgzi_1QAAAAJ">GoogleScholar</a></div>
  <div class="menu-item"><a href="https://orcid.org/0000-0001-5345-6982">ORCID</a></div>
  </td>
  <td id="layout-content">
  <div id="toptitle">


<h1>Publications and Submissions </h1>
</div>

<h4><a href="https://scholar.google.com/citations?hl=en&user=pgzi_1QAAAAJ">Google Scholar</a> </h4> Seven first-author and five senior-author publications.   



<h3>Publications and Submissions</h3>

<ul>

<li> S. Majidian, S. Hwang, M. Zakeri, B Langmead.  <a href="https://doi.org/10.1093/bib/bbaf267">EvANI benchmarking workflow for evolutionary distance estimation. </a> Briefings in Bioinformatics, 2025.
 </li>

   <li> S. Majidian,  Y. Nevers, A. Yazdizadeh, A. Vesztrocy, S. Pascarelli, D. Moi, N. Glover, A. Altenhoff, C. Dessimoz. <a href="https://doi.org/10.1038/s41592-024-02552-8">
      Orthology inference at scale with FastOMA. </a>, Nature Methods, 2025. 
      <a href="https://github.com/DessimozLab/fastoma">code</a>,  <a href="https://www.youtube.com/watch?v=KGetTUMDvlA&t=2s"> YouTube Talk.</a>,
 </li>


 <li> K. Vaddadi, M. Lin, S. Majidian, T. Mun, B. Langmead.<a href="https://doi.org/10.1101/2023.11.30.568362">
       Minimizing reference bias with an imputed personalized reference. </a>,  Submitted 2025.
 </li>

 <li> S. Tan, S. Majidian, B Langmead, M. Zakeri.<a href="https://doi.org/10.1101/2025.05.22.655637">
        Movi Color: fast and accurate long-read classification with the move structure. </a>,  Submitted 2025.
 </li>


 <li> S.A., Malekpour, A. Kalirad, S. Majidian* <a href="https://doi.org/10.1093/gbe/evaf050">
      Inferring the selective history of CNVs using a maximum likelihood model. </a>, Genome Biology and Evolution. 2025.
 </li>

   <li>N. Dwarshuis, N. Kalra, ..., J. Wagner, S. Majidian*, J. Zook*. 
    <a href="https://www.nature.com/articles/s41467-024-53260-y">
      The GIAB genomic stratifications resource for human reference genomes. </a>, Nature Communications, 2024. 
 </li>

  <li> D. Dylus , A. Altenhoff , S. Majidian , F. Sedlazeck, C. Dessimoz, 
    <a href="https://www.nature.com/articles/s41587-023-01753-4">
      Inference of phylogenetic trees directly from raw sequencing reads using Read2Tree</a>,
      Nature Biotechnology. 2024.
    <a href="https://github.com/DessimozLab/read2tree">code</a>.
  </li>

 <li> M. Ndiaye, S. Banos, L. Fitzgerald, A. Yazdizadeh, S. Oreshkov, C. Dessimoz, F. Sedlazeck, N. Glover, S. Majidian*. 
    <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03414-4">
      When less is more: applications of minimizers in genomics</a>,
     Genome Biology. 2024.
  </li>

  <li>  S. Lamei, A. Beyraghdar, A. Shojaeiyan, S. Majidian.
    <a href="https://doi.org/10.1186/s12864-024-10253-x">
      Transcriptomic data reveals the dynamics of terpenoids biosynthetic pathway of fenugreek. </a>, 
      BMC Genomics, 2024. 
  </li>

  <li> A. Altenhoff, A.W Vesztrocy,C. Bernard, C. Train, A. Nicheperovich, S. PrietoBanos, I. Julca, D. Moi,Y. Nevers, S. Majidian, C. Christophe Dessimoz & N. Glover, 
    <a href="https://doi.org/10.1093/nar/gkad1020">
      OMA orthology in 2024: improved prokaryote coverage, ancestral and extant GO enrichment, a revamped synteny viewer and more in the OMA Ecosystem. </a> 
      Nucleic Acids Research. 2024.
      <a href="https://omabrowser.org/">Web browser</a>.
  </li>


  <li>  F. Langschied, .., S. Majidian, .., I. Ebersberger
    <a href="https://academic.oup.com/gbe/advance-article/doi/10.1093/gbe/evae224/7822254">
      Quest for Orthologs in the Era of Biodiversity Genomics. </a>, 
      Genome Biology and Evolultion, 2024. 
  </li>


  <li> S. Majidian, D. Paiva Agustinho,  F. Sedlazeck, M. Mahmoud, 
    <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03061-1">
      Genomic variant benchmark: if you cannot measure it, you cannot improve it.</a> 
      Genome Biology. 2023.
  </li>

<li>  A. Altenhoff ... S. Majidian.  I. Ebersberger, E. Sonnhammer.
  <a href="https://doi.org/10.1093/nargab/lqae167">
      New developments for the Quest for Orthologs Benchmark Service. </a>, NAR Genomics and Bioinformatics.
 </li>



  <li> <a href="https://f1000research.com/articles/12-382/v1 ">
      Phylogenetic placement of whole genome duplications in yeasts through quantitative analysis of hierarchical orthologous groups.  </a>. F1000 preprint.
  </li>


  <li> P. Liakopoulous, B. Banfalvi, X. Wang, S. Majidian, T. Farias, C. Dessimoz and A. Claudia-Sima. 
  Towards predicting essential proteins via federated SPARQL queries, 
  <a href="https://github.com/dssib/swat23-data-in-use"> Code</a>,
  <a href="https://ceur-ws.org/Vol-3415/paper-45.pdf"> Conference proceedings</a>,
    The Semantic Web Applications and Tools for Healthcare and Life Science conference (SWAT4HCLS), Basel, Switzerland 2023. 
  </li>


  <li> A. Nicheperovich, A. Altenhoff, C. Dessimoz, S. Majidian*, 
    <a href="https://doi.org/10.1093/bioinformatics/btac163">
    OMAMO: orthology-based alternative model organism selection</a>,
    Bioinformatics, 2022,
   <a href="https://github.com/DessimozLab/omamo">code</a>,
   <a href="https://omamo.omabrowser.org/">web service</a>.
  </li>

  <li> S. Majidian., M. Mohades, M. Kahaei, 
    <a href="https://www.sciencedirect.com/science/article/pii/S1051200420302256">
    Matrix completion with weighted constraint for haplotype estimation</a>,
    Digital Signal Processing, 2021.
  </li>

  <li> S. Majidian, F. Sedlazeck,
    <a href="https://academic.oup.com/gigascience/article/9/7/giaa078/5875849">
    PhaseME: Automatic rapid assessment of phasing quality and phasing improvement</a>,
    GigaScience, 2020,
    <a href="https://github.com/sinamajidian/phaseme/">code</a>.
  </li>

  <li> S. Majidian, M. Kahaei, D. de Ridder, 
    <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03584-5">
    Hap10: reconstructing accurate and long polyploid haplotypes using linked reads</a>,
    BMC Bioinformatics, 2020,
    <a href="https://github.com/sinamajidian/Hap10/">code</a>.
  </li>

  <li> S. Majidian, M. Kahaei, D. de Ridder,
    <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234470">
    Minimum error correction-based haplotype assembly: considerations for long read data</a>,
    PLOS one, 2020,
    <a href="https://github.com/sinamajidian/Hap10/">code</a>.
  </li>
 
  <li>  M. Mohades,  S. Majidian, M. Kahaei,
    <a href="https://ieeexplore.ieee.org/abstract/document/8686170/">
    Haplotype assembly using manifold optimization and error correction mechanism</a>,
    IEEE Signal Processing Letters, 2019, paper.
  </li>

  <li> S. Majidian, M. Kahaei,
    <a href="https://doi.org/10.1371/journal.pone.0214455/">
    NGS based haplotype assembly using matrix completion</a>,
    PLOS one, 2019,
    <a href="https://github.com/sinamajidian/MEC">code</a>.
  </li>

  <li> R. Ezazi, M. ..., S. Majidian, E. Stefani, M. Pindo, C. Donati,
    <a href="https://doi.org/10.1093/femsec/fiab107/">
    Responses of cucumber (Cucumis sativus L.) rhizosphere microbial community to some agronomic management practices</a>,
    FEMS Microbiology Ecology, 2021.
  </li>

  <li> E. Valipour, ...,  S. Majidian, F. Motevaseli, M. Modarressi,
    <a href="https://doi.org/10.1016/j.gene.2020.144856/">
    Anti-angiogenic effects of testis-specific gene antigen 10 on primary endothelial cells</a>,
    Gene Elsevier, 2020.
  </li>

  <li> A. Ahmadi, S. Majidian, M. Kahaei,
    <a href="https://doi.org/10.1007/s00034-020-01613-5/">
    Matrix completion using graph total variation based on directed laplacian matrix</a>,
    Circuits, Systems, and Signal Processing Springer, 2021.
  </li>

  <li> K. Walker, ..., S. Majidian, ..., Fritz J Sedlazeck, Ben Busby, 
    <a href="https://f1000research.com/articles/11-530">
      The third international hackathon for applying insights into large-scale genomic composition to use cases in a wide range of organisms</a>,
      F1000Research, 2022.
    <a href="https://github.com/collaborativebioinformatics/">code</a>.
  </li>

</ul>




<h3> Invited talks and presentations</h3>


<li> ANI, are you OK? Challenges for k-mer-based estimation of evolutionary distances. 
  <a href="https://github.com/sinamajidian/EvANI"> Pipeline.</a>,
  lightning talk, Biological Data Science 2024, Cold Spring Harbor Laboratory, 14 Nov 2024. US. 
  </li>
  

<li>  Orthology inference at scale with FastOMA,
    Workshop on Emerging Methods for Sequence Analysis WEMSA, Pennsylvania State University. US.
  </li>

  <li> Computational comparative genomics with FastOMA and read2tree,
    Institut Pasteur. March 2024, France. 
  </li>

  <li> Inference of phylogenetic trees directly from raw sequencing reads using Read2Tree,
    Department of Microbiology, University of Lausanne. May 2023, Switzerland. 
  </li>

  <li> Computational Comparative Genomics in the era of BioGenome projects,
  <a href="https://github.com/sinamajidian/workshops/blob/main/talks/ComputationalComparativeGenomics_Zurich_Nov6.pdf"> Slides.</a>,
    University of Zurich. 23 Nov 2023, Switzerland. 
  </li>

  <li> Computational Comparative Genomics in the era of BioGenome projects,
  <a href="https://github.com/sinamajidian/workshops/blob/main/talks/ComputationalComparativeGenomics_Zurich_Nov6.pdf"> Slides.</a>,
    University of Zurich. 23 Nov 2023, Switzerland. 
  </li>


  <li> Orthology inference at scale with FastOMA,
  <a href="https://www.youtube.com/watch?v=KGetTUMDvlA&t=2s"> YouTube Video.</a>,
  <a href="https://github.com/sinamajidian/workshops/blob/main/talks/fastOMA_ismb_2023.pdf"> Slides.</a>,
    International Conference on Intelligent Systems for Molecular Biology. 23-27 July 2023. France.
  </li>


  <li> BIOQA: toward a representative benchmark dataset of biological questions/answers involving orthology, gene expression, and complementary omics data.,
  <a href="https://github.com/sinamajidian/workshops/blob/main/talks/bioqa.pdf"> Slides.</a>,
    21st European Conference on Computational Biology. Quest For Orthologs session. 12-21 September 2022. Spain.
  </li>


  <li>A fast pipeline for species tree inference using placement in Hierarchical Orthologous Groups,
  <a href="https://github.com/smajidian/workshops/raw/main/talks/fastoma.pdf"> Slides.</a>,
    Biodiversity Genomics 2021, a virtual conference organised by Wellcome Trust Genome Campus. Aug. 2021.
  </li>

  <li> PhaseRE: Identifying recombination location per sample using long reads,
  <a href="https://github.com/smajidian/workshops/raw/main/talks/phasere.pdf"> Poster.</a>,
  BC2 Basel Computational Biology Conference, Switzerland. Sep. 2021.
  </li>

  <li> Towards pathway-specific model organism with orthology relationships.
    <a href="https://github.com/smajidian/workshops/raw/main/talks/omamo.pdf"> PDF of presentation.</a>,
    International Conference on Genetics and Human Genomics, Iran. Dec. 2021.
  </li>

  <li> Haplotype assembly using rank minimization. The 7th Conference on Bioinformatics, Iran, Jan. 2018.
  </li>

  <li> Haplotype estimation for NGS data using polynomial optimization. The 10th Biotechnology Congress, Iran, Aug. 2017.
  </li>

  </ul>




</td>
</tr>
</table>
</body>
</html>
