<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>

  <!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-Y9EJC8CJW6"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-Y9EJC8CJW6');
</script>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
<title> Sina Majidian's Publications </title>
</head>
<body>
  <table summary="Table for page layout." id="tlayout">
  <tr valign="top">
  <td id="layout-menu">
  <div class="menu-category">Home</div>
  <div class="menu-item"><a href="index.html" class="current">About&nbsp;me</a></div>
  <div class="menu-item"><a href="publications.html">Publications</a></div>
  <div class="menu-item"><a href="teaching.html" class="current">Teaching</a></div>
  <div class="menu-item"><a href="research.html">Research</a></div>
  <div class="menu-item"><a href="blog.html">Blog</a></div>
  <div class="menu-item"><a href="blog_persian.html">وبلاگ فارسی</a></div>
  <div class="menu-category">Social Media</div>
  <div class="menu-item"><a href="https://www.linkedin.com/in/sina-majidian/">Linkedin</a></div>
  <div class="menu-item"><a href="https://twitter.com/SinaMajidian">Twitter</a></div>
  <div class="menu-category">Links</div>
  <div class="menu-item"><a href="https://github.com/sinamajidian/">GitHub</a></div>
  <div class="menu-item"><a href="https://scholar.google.nl/citations?user=pgzi_1QAAAAJ">GoogleScholar</a></div>
  <div class="menu-item"><a href="https://orcid.org/0000-0001-5345-6982">ORCID</a></div>
  </td>
  <td id="layout-content">
  <div id="toptitle">


<h1>Publications and Submissions </h1>
</div>

<h4><a href="https://scholar.google.com/citations?hl=en&user=pgzi_1QAAAAJ">Google Scholar</a> </h4>

<h3>Publications</h3>

<ul>

  <li> D. Dylus , A. Altenhoff , S. Majidian , F. Sedlazeck, C. Dessimoz., 
    <a href="https://www.biorxiv.org/content/10.1101/2022.04.18.488678">
      Read2Tree: scalable and accurate phylogenetic trees from raw reads</a>,
      Accepted in Nature Biotechnology. 2023.
    <a href="https://github.com/DessimozLab/read2tree">code</a>.
  </li>

  <li> A. Nicheperovich, A. Altenhoff, C. Dessimoz, <b>S. Majidian*</b>., 
    <a href="https://doi.org/10.1093/bioinformatics/btac163">
    OMAMO: orthology-based alternative model organism selection</a>,
    Bioinformatics, 2022,
   <a href="https://github.com/DessimozLab/omamo">code</a>,
   <a href="https://omamo.omabrowser.org/">web service</a>.
  </li>

  <li> <b>S. Majidian</b>., M. Mohades, M. Kahaei., 
    <a href="https://www.sciencedirect.com/science/article/pii/S1051200420302256">
    Matrix completion with weighted constraint for haplotype estimation</a>,
    Digital Signal Processing, 2021.
  </li>

  <li> <b>S. Majidian</b>, F. Sedlazeck.,
    <a href="https://academic.oup.com/gigascience/article/9/7/giaa078/5875849">
    PhaseME: Automatic rapid assessment of phasing quality and phasing improvement</a>,
    GigaScience, 2020,
    <a href="https://github.com/sinamajidian/phaseme/">code</a>.
  </li>

  <li> <b>S. Majidian</b>, M. Kahaei, D. de Ridder., 
    <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03584-5">
    Hap10: reconstructing accurate and long polyploid haplotypes using linked reads</a>,
    BMC Bioinformatics, 2020,
    <a href="https://github.com/sinamajidian/Hap10/">code</a>.
  </li>

  <li> <b>S. Majidian</b>, M. Kahaei, D. de Ridder.,
    <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234470">
    Minimum error correction-based haplotype assembly: considerations for long read data</a>,
    PLOS one, 2020,
    <a href="https://github.com/sinamajidian/Hap10/">code</a>.
  </li>
 
  <li>  M. Mohades,  <b>S. Majidian</b>, M. Kahaei.,
    <a href="https://ieeexplore.ieee.org/abstract/document/8686170/">
    Haplotype assembly using manifold optimization and error correction mechanism</a>,
    IEEE Signal Processing Letters, 2019, paper.
  </li>

  <li> <b>S. Majidian</b>, M. Kahaei.,
    <a href="https://doi.org/10.1371/journal.pone.0214455/">
    NGS based haplotype assembly using matrix completion</a>,
    PLOS one, 2019,
    <a href="https://github.com/sinamajidian/MEC">code</a>.
  </li>

  <li> R. Ezazi, M. ..., S. Majidian, E. Stefani, M. Pindo, C. Donati.,
    <a href="https://doi.org/10.1093/femsec/fiab107/">
    Responses of cucumber (Cucumis sativus L.) rhizosphere microbial community to some agronomic management practices</a>,
    FEMS Microbiology Ecology, 2021.
  </li>

  <li> E. Valipour, ...,  <b>S. Majidian</b>, F. Motevaseli, M. Modarressi.,
    <a href="https://doi.org/10.1016/j.gene.2020.144856/">
    Anti-angiogenic effects of testis-specific gene antigen 10 on primary endothelial cells</a>,
    Gene Elsevier, 2020.
  </li>

  <li> A. Ahmadi, <b>S. Majidian*</b>, M. Kahaei*.,
    <a href="https://doi.org/10.1007/s00034-020-01613-5/">
    Matrix completion using graph total variation based on directed laplacian matrix</a>,
    Circuits, Systems, and Signal Processing Springer, 2021.
  </li>

  <li> K. Walker, ..., <b>S. Majidian</b>, ..., Fritz J Sedlazeck, Ben Busby., 
    <a href="https://f1000research.com/articles/11-530">
      The third international hackathon for applying insights into large-scale genomic composition to use cases in a wide range of organisms</a>,
      F1000Research, 2022.
    <a href="https://github.com/collaborativebioinformatics/">code</a>.
  </li>

</ul>

<h3>Submissions & Drafts </h3>

<ul>

  <li> Genomic Benchmarks: If you cannot measure it, you cannot improve it. 2023.
  </li>

  <li> When less is more: applications of minimizers in genomics. 2023.
  </li>

  <li> Phylogenetic placement of whole genome duplications in yeasts through quantitative analysis of hierarchical orthologous groups. 2022.
  </li>

  <li> RHOXF1 is a candidate member of the transcriptional regulatory complex of TSGA10 in testis and cancer. 2020.
  </li>

  <li> Overexpression of TSGA10 in HeLa cells downregulates miR-183 level; TSGA10 defeats miR-183/HIF1A network. 2019.
  </li>


</ul>


<h3> Invited talks and presentations</h3>

  <ul> P. Liakopoulous, B. Banfalvi, X. Wang,  C. Dessimoz., <b> S. Majidian</b>  and A. Claudia-Sima. 
  <li>BIOQA: toward a representative benchmark dataset of biological questions/answers involving orthology, gene expression, and complementary omics data.,
  <a href="https://github.com/sinamajidian/workshops/blob/main/talks/bioqa.pdf"> PDF of presentation.</a>,
    21st European Conference on Computational Biology. Quest For Orthologs session. 12-21 September 2022. Spain.
  </li>

  <ul> P. Liakopoulous, B. Banfalvi, X. Wang, S. Majidian, T. Farias, C. Dessimoz and A. Claudia-Sima. 
  <li>Towards predicting essential proteins via federated SPARQL queries.  ,
  <a href="https://github.com/dssib/swat23-data-in-use"> Poster presentation..</a>,
    The Semantic Web Applications and Tools for Healthcare and Life Science conference (SWAT4HCLS), Basel, Switzerland 2023. 
  </li>


  <li>A fast pipeline for species tree inference using placement in Hierarchical Orthologous Groups,
  <a href="https://github.com/smajidian/workshops/raw/main/talks/fastoma.pdf"> PDF of presentation.</a>,
    Biodiversity Genomics 2021, a virtual conference organised by Wellcome Trust Genome Campus. Aug. 2021.
  </li>

  <li>PhaseRE: Identifying recombination location per sample using long reads,
  <a href="https://github.com/smajidian/workshops/raw/main/talks/phasere.pdf"> PDF of poster.</a>,
  BC2 Basel Computational Biology Conference, Switzerland. Sep. 2021.
  </li>

  <li> Towards pathway-specific model organism with orthology relationships.
    <a href="https://github.com/smajidian/workshops/raw/main/talks/omamo.pdf"> PDF of presentation.</a>,
    International Conference on Genetics and Human Genomics, Iran. Dec. 2021.
  </li>

  <li> Haplotype assembly using rank minimization. The 7th Conference on Bioinformatics, Iran, Jan. 2018.
  </li>

  <li> Haplotype estimation for NGS data using polynomial optimization. The 10th Biotechnology Congress, Iran, Aug. 2017.
  </li>

  </ul>




</td>
</tr>
</table>
</body>
</html>
