<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>

  <!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-Y9EJC8CJW6"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-Y9EJC8CJW6');
</script>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
<title>Sina Majidian's Research</title>
<meta name="description" content="Data science how is used in biology using programming languages python and R and C I work in the field of Bioinformatics, computational biologist, Comparative genomics. ">
</head>
<body>
  <table summary="Table for page layout." id="tlayout">
  <tr valign="top">
  <td id="layout-menu">
  <div class="menu-category">Home</div>
  <div class="menu-item"><a href="index.html" class="current">About&nbsp;me</a></div>
  <div class="menu-item"><a href="publications.html">Publications</a></div>
  <div class="menu-item"><a href="teaching.html" class="current">Teaching</a></div>
  <div class="menu-item"><a href="research.html">Research</a></div>
  <div class="menu-item"><a href="news.html">News</a></div>
  <div class="menu-category">Social Media</div>
  <div class="menu-item"><a href="https://www.linkedin.com/in/sina-majidian/">Linkedin</a></div>
  <div class="menu-item"><a href="https://bsky.app/profile/sinamajidian.bsky.social">BlueSky</a></div>  
  <div class="menu-item"><a href="https://twitter.com/SinaMajidian">Twitter</a></div>
  <div class="menu-category">Links</div>
  <div class="menu-item"><a href="https://github.com/sinamajidian/">GitHub</a></div>
  <div class="menu-item"><a href="https://scholar.google.nl/citations?user=pgzi_1QAAAAJ">GoogleScholar</a></div>
  <div class="menu-item"><a href="https://orcid.org/0000-0001-5345-6982">ORCID</a></div>
  </td>
  <td id="layout-content">
  <div id="toptitle">


<h1>Dr. Sina Majidian's Research Studies</h1>
</div>


Pangenomes are now redefining our understanding of genetic variations across populations and genome evolution across species. A single reference genome cannot fully represent human genetic diversity, even when it is complete enough to be considered “Telomere-to-Telomere”. To fully harness the power of pangenomes in biomedicine, there is a pressing need for efficient methods to store, visualize, and extract relevant information.  My research program aims to understand human genome variation and evolution across different genomic regions by developing interpretable and efficient methods in comparative pan-genomics, leveraging machine learning methods and statistical analysis. 


<p>&nbsp;</p> 



<center><img src="photos/faculty_mindmap_past.png" alt="Past Research studies of Dr. Sina Majidian, Faculty candidate" width="800px" />&nbsp; </center>


<p>&nbsp;</p> 


<h3>  Comparative genomics </h3>


Orthology and phylogeny inference at scale, along with their applications, are my main research interests. I have developed methods and written software in the field of comparative genomics, aiming to enable evolutionary analysis at the scale of the Tree of Life. I contributed to the development of Read2Tree, a fast and accurate method for inferring phylogenies from sequencing reads. I am also the first author of FastOMA, a tool for accurately identifying orthologous genes by distinguishing them from paralogs. This tool makes a significant impact in orthology prediction and addresses the challenges of growing large-scale genomic data. Orthology information can also be leveraged to align single-cell data across different species.


<ul>
  <li  style="color:black;font-size:14px" > S. Majidian,  Y. Nevers, A. Yazdizadeh, A. Vesztrocy, S. Pascarelli, D. Moi, N. Glover, A. Altenhoff, C. Dessimoz. Orthology inference at scale with FastOMA.
   <a href="https://doi.org/10.1038/s41592-024-02552-8"> Nature Methods</a>,  2024. 
      <a href="https://github.com/DessimozLab/fastoma">code</a>,  <a href="https://www.youtube.com/watch?v=KGetTUMDvlA&t=2s"> YouTube Talk.</a>,
 </li>

   <li style="color:black;font-size:14px"> D. Dylus , A. Altenhoff , S. Majidian , F. Sedlazeck, C. Dessimoz, Inference of phylogenetic trees directly from raw sequencing reads using Read2Tree.
    <a href="https://www.nature.com/articles/s41587-023-01753-4"> Nature Biotechnology</a>,
       2024.
    <a href="https://github.com/DessimozLab/read2tree">code</a>.
  </li>

 <li style="color:black;font-size:14px"> A. Nicheperovich, A. Altenhoff, C. Dessimoz, S. Majidian*,  OMAMO: orthology-based alternative model organism selection.
    <a href="https://doi.org/10.1093/bioinformatics/btac163"> Bioinformatics</a>,
     2022,
   <a href="https://github.com/DessimozLab/omamo">code</a>,
   <a href="https://omamo.omabrowser.org/">web service</a>.
  </li>


   <li style="color:black;font-size:14px"> D A. Yazdizadeh, A. Altenhoff, N. Romashchenko, C. Dessimoz, S. Majidian*, OrthoXML-tools: a toolkit for manipulating OrthoXML files for orthology data.
    <a href="https://github.com/DessimozLab/orthoxml-tools">code</a>.
  </li>



 </ul>

<center><img src="photos/Orthology.png" alt="Orthology inference" width="800px" />&nbsp;</center>


<p>&nbsp;</p> 

<h3> Genomic variant benchmark and personalized human pangenome</h3>


Genomic variant benchmark datasets are crucial for assessing the performance of sequencing technologies and analytical methods; however, their development involves multiple sequencing technologies, different variant calling tools, and laborious manual curation. In one study, I reviewed available benchmark datasets and their utility, with a focus on genes of medical relevance. In another study, in collaboration with the Genome in a Bottle team, I co-led an effort to develop a genomic resource by defining various challenging genomic contexts used to investigate the strengths and weaknesses of variant callers, which was published in Nature Communications.


<ul>
<li style="color:black;font-size:14px"> N. Dwarshuis, N. Kalra, ..., J. Wagner, S. Majidian*, J. Zook*, The GIAB genomic stratifications resource for human reference genomes.  <a href="https://www.nature.com/articles/s41467-024-53260-y"> Nature Communications</a>, 2024. 
</li>

<li style="color:black;font-size:14px"> S. Majidian, D. Paiva Agustinho,  F. Sedlazeck, M. Mahmoud, Genomic variant benchmark: if you cannot measure it, you cannot improve it.
  <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-03061-1"> Genome Biology</a>, 2023.
</li>

<li style="color:black;font-size:14px"> K. Vaddadi, M. Lin, S. Majidian, T. Mun, B. Langmead.
       Minimizing reference bias with an imputed personalized reference<a href="https://doi.org/10.1101/2023.11.30.568362"> preprint</a> 2025.
</li>



<li style="color:black;font-size:14px">F. Palizban, S. Majidian* 
       cfCNV: an optimized pipeline for detecting copy number variations for cell free DNA data. <a href="https://www.nature.com/articles/s41431-024-01734-4#Sec2058">European Journal of Human Genetics</a>,  57th ESHG European Society of Human Genetics. 2025.
</li>


 </ul>

<center><img src="photos/Benchmark.png" alt="Genetic variation Benchmark" width="412px"/>&nbsp;</center>



<p>&nbsp;</p> 


<h3>  Haplotype assembly</h3>


 Haplotype assembly was the main focus of my PhD, leading to five first-author and two second-author publications. I have developed software for estimating haplotype blocks from single nucleotide variants (SNVs) called from DNA sequencing reads. In one project, I benefited from low-rank matrix recovery in haplotype estimation and applied it to human sequencing data. In another project, I studied hexaploid sweet potato (Ipomoea batatas) using a 10X Genomics linked-read dataset, which resulted in long and accurate haplotypes. A longstanding problem in the field was understanding the limitations of the Minimum Error Correction (MEC) approach in haplotype assembly, for which I developed a solid framework. 


<ul>


  <li style="color:black;font-size:14px"> S. Majidian, F. Sedlazeck, PhaseME: Automatic rapid assessment of phasing quality and phasing improvement.
    <a href="https://academic.oup.com/gigascience/article/9/7/giaa078/5875849"> GigaScience</a>,2020, <a href="https://github.com/sinamajidian/phaseme/">code</a>.
  </li>

  <li style="color:black;font-size:14px"> S. Majidian, M. Kahaei, D. de Ridder, Hap10: reconstructing accurate and long polyploid haplotypes using linked reads.
    <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03584-5"> BMC Bioinformatics</a>, 2020,
    <a href="https://github.com/sinamajidian/Hap10/">code</a>.
  </li>

  <li style="color:black;font-size:14px"> S. Majidian, M. Kahaei, D. de Ridder, Minimum error correction-based haplotype assembly: considerations for long read data.
    <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0234470"> PLOS one</a>, 2020, <a href="https://github.com/sinamajidian/Hap10/">code</a>.
  </li>
 
  <li style="color:black;font-size:14px">  M. Mohades,  S. Majidian, M. Kahaei, Haplotype assembly using manifold optimization and error correction mechanism.
    <a href="https://ieeexplore.ieee.org/abstract/document/8686170/"> IEEE Signal Processing Letters</a>, 2019.
  </li>

</ul>

<center><img src="photos/Haplotype.png" alt="Haplotype inference" width="412px" />&nbsp;</center>


<p>&nbsp;</p> 


<h3> Bacterial pangenome and metagenomics classification </h3>


Metagenomics is the study of genetic material recovered directly from specific environments, such as the gut microbiota or soil. High-throughput DNA sequencing technologies revolutionized the field by enabling large-scale metagenomic studies. Initially, these technologies produced short reads of 100-250 nucleotides, but recent advances in long-read sequencing allow for the capture of thousands of bases with high accuracy, providing richer information for analysis. By sequencing DNA samples, we can characterize microbial biodiversity and determine which bacterial strains or species are present, a task known as metagenomic classification.


<ul>
<li style="color:black;font-size:14px"> S. Majidian,  B. Langmead.  MetaKpick: machine learning–based metagenomic classification with multi k-mer-based pangenome indexes, 2024. 
</li>

<li style="color:black;font-size:14px"> S. Tan, S. Majidian, B. Langmead, M. Zakeri.
        Movi Color: fast and accurate long-read classification with the move structure<a href="https://doi.org/10.1101/2025.05.22.655637"> ACM-BCB</a>,  2025.
</li>

 </ul>






</ul>

<h2>Supervision and mentorship</h2>


<h4> Graduate students </h4>

<ul>


<li> Fahimeh Palizban. Inferring copy number variation with cell-free DNA data, University of Lausanne, 2024.   (Currently at Children's Hospital of Philadelphia)
  </li>

 <li> Arun Maurya. Reconstructing cancer phylogenies with Spatial Transcriptomics, University of Lausanne, 2023.  (Currently Research Engineer at Institut Pasteur)
  </li> 
 
  <li> Sara Lamei. Analysing biosynthetic pathway of terpenoids in fenugreek using RNA-seq data. Tarbiat Modares University, 2022. (Currently at Saarland University)
  </li>
 
 <li> Rajarshi Mondal. T2T gene annotations of the Major Histocompatibility Complex. Pondicherry University, 2022.
  </li>
  
  <li> Samuel Moix. Detecting whole genome duplication using hierarchical orthologous groups, University of Lausanne, 2021. (Now PhD student at the University of Lausanne, Switzerland.)
  </li>

  <li> Mathijs van Kooten. Creating an ensemble method combining haplotype estimates. Wageningen university, 2019.
  </li>

</ul>


<h4>Summer interns </h4>
<ul>

  <li> Ali Yazdizadeh. Importance of phylogenetic rooting for orthology inference, 2022.  (Incoming PhD student at Carnegie Mellon University, USA.)
  </li>

  <li>  Borbala Banfalvi, Natural Language Question Answering over Knowledge Graphs, 2022. (Currently PhD student at Queen Mary University London).
  </li>

<li>  Claire Wang, Natural Language Question Answering over Knowledge Graphs,  2022. (Currently PhD student at University of Cambridge).
  </li>


  <li> Alina Nicheperovich. Orthology guided model organism selection, 2021. (Now DPhil Candidate at the University of Oxford, UK.)
  </li>


</ul>


</td>
</tr>
</table>
</body>
</html>
