<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>

  <!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-Y9EJC8CJW6"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-Y9EJC8CJW6');
</script>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
<title>Sina Majidian's Research</title>
<meta name="description" content="Data science how is used in biology using programming languages python and R and C I work in the field of Bioinformatics, computational biologist, Comparative genomics. ">
</head>
<body>
  <table summary="Table for page layout." id="tlayout">
  <tr valign="top">
  <td id="layout-menu">
  <div class="menu-category">Home</div>
  <div class="menu-item"><a href="index.html" class="current">About&nbsp;me</a></div>
  <div class="menu-item"><a href="publications.html">Publications</a></div>
  <div class="menu-item"><a href="teaching.html" class="current">Teaching</a></div>
  <div class="menu-item"><a href="research.html">Research</a></div>
  <div class="menu-item"><a href="news.html">News</a></div>
  <div class="menu-category">Social Media</div>
  <div class="menu-item"><a href="https://www.linkedin.com/in/sina-majidian/">Linkedin</a></div>
  <div class="menu-item"><a href="https://bsky.app/profile/sinamajidian.bsky.social">BlueSky</a></div>  
  <div class="menu-item"><a href="https://twitter.com/SinaMajidian">Twitter</a></div>
  <div class="menu-category">Links</div>
  <div class="menu-item"><a href="https://github.com/sinamajidian/">GitHub</a></div>
  <div class="menu-item"><a href="https://scholar.google.nl/citations?user=pgzi_1QAAAAJ">GoogleScholar</a></div>
  <div class="menu-item"><a href="https://orcid.org/0000-0001-5345-6982">ORCID</a></div>
  </td>
  <td id="layout-content">
  <div id="toptitle">


<h1>Past Research Studies</h1>
</div>


<li> Orthology and phylogeny inference at scale, along with their applications, are my main research interests. I have developed methods and written software in the field of comparative genomics, aiming to enable evolutionary analysis at the scale of the Tree of Life. I contributed to the development of Read2Tree, a fast and accurate method for inferring phylogenies from sequencing reads. I am also the first author of FastOMA, a tool for accurately identifying orthologous genes by distinguishing them from paralogs. This tool makes a significant impact in orthology prediction and addressing the challenges of growing large- scale genomic data.  
  </li>

<img src="photos/Orthology.png" alt="Orthology inference" width="800px" />&nbsp;


<li> 
 Haplotype assembly was the main focus of my PhD, leading to five first-author and two second-author publications. I have developed software for estimating haplotype blocks from single nucleotide variants (SNVs) called from DNA sequencing reads. In one project, I benefited from low-rank matrix recovery in haplotype estimation and applied it to human sequencing data. In another project, I studied hexaploid sweet potato (Ipomoea batatas) using a 10X Genomics linked-read dataset, which resulted in long and accurate haplotypes. A longstanding problem in the field was understanding the limitations of the Minimum Error Correction (MEC) approach in haplotype assembly, for which I developed a solid framework. 
  </li>

<img src="photos/Haplotype.png" alt="Haplotype inference" width="412px" />&nbsp;


<li> 
Genomic variant benchmark datasets are crucial for assessing the performance of sequencing technologies and analytical methods; however, their development involves multiple sequencing technologies, different variant calling tools, and laborious manual curation. In one study, I reviewed available benchmark datasets and their utility, with a focus on genes of medical relevance. In another study, in collaboration with the Genome in a Bottle team, I co-led an effort to develop a genomic resource by defining various challenging genomic contexts used to investigate the strengths and weaknesses of variant callers, which is published in Nature Communication.
  </li>

<img src="photos/Benchmark.png" alt="Genetic variation Benchmark" width="412px"/>&nbsp;



 <br>


</ul>

<h2>Supervision and mentorship</h2>


<h4> Graduate students </h4>

<ul>


<li> Fahimeh Palizban. Inferring copy number variation with cell-free DNA data, University of Lausanne, 2024.  
  </li>

 <li> Arung Maurya. Reconstructing cancer phylogenies with Spatial Transcriptomics, University of Lausanne, 2023.  
  </li>
 
  <li> Sara Lamei. Analysing biosynthetic pathway of terpenoids in fenugreek using RNA-seq data. Tarbiat Modares University, 2022.
  </li>
 
 <li> Rajarshi Mondal.  T2T gene annotations of the Major Histocompatibility Complex. Pondicherry University, 2022.
  </li>
  

  <li> Samuel Moix. Detecting whole genome duplication using hierarchical orthologous groups, University of Lausanne, 2021. (Now PhD student at the University of Lausanne, Switzerland.)
  </li>

  <li> Mathijs van Kooten. Creating an ensemble method combining haplotype estimates. Wageningen university, 2019. (Now at Twintos, the Netherlands).
  </li>


</ul>


<h4>Summer interns </h4>
<ul>

  <li> Ali Yazdizadeh. Importance of phylogenetic rooting for orthology inference, 2022. (Incoming PhD student in Computational Biology, USA.)
  </li>

  <li>  Borbala Banfalvi, Claire Wang, and Petros Liakopoulos.  Natural Language Question Answering over Knowledge Graphs,  2022.
  </li>

  <li> Alina Nicheperovich. Orthology guided model organism selection, 2021. (Now DPhil Candidate at the University of Oxford, UK.)
  </li>


</ul>


</td>
</tr>
</table>
</body>
</html>
