---
title: 'Graph in bioinformatics'
date: 2021-07-01
permalink: /posts/2021/07/graph-bioinformatics/
tags:
  - graph
  - bioinformatics
  - pangenome
---


Recently, I have started to read on the applications of graph analysis in bioinformatics. Firstly, I gathered some articles. 

 
Review articles 
------

- Computational pan-genomics status, promises and challenges [paper](https://doi.org/10.1093/bib/bbw089).

- Pangenome Graphs [paper](https://doi.org/10.1146/annurev-genom-120219-080406).



Benedict Paten & Erik Garrison
------
- Genome graphs and the evolution of genome inference - Genome Graphs- A graph extension of the positional Burrows–Wheeler transform and its applications- A graph- based approach to diploid genome assembly- Efficient dynamic variation graphs- Thesis, Graphical pangenomics
- Genotyping common, large structural variations in 5,202 genomes using pangenomes, the Giraffe mapper, and the vg toolkit- GRAFIMO-  variant and haplotype aware motif scanning on pangenome graphs -  Haplotype- aware graph indexes- Haplotype- aware pantranscriptome analyses using spliced pangenome graphs- Nanopore sequencing and the Shasta toolkit enable efficient de novo assembly of eleven human genomes- Removing reference bias and improving indel calling in ancient DNA data analysis by mapping to a sequence variation graph- Succinct dynamic variation graphs - Superbubbles, Ultrabubbles, and Cacti - The pluralistic promise of pangenome graphs @ WABI 2020- Variation graph toolkit improves read mapping by representing genetic variation in the reference 



Tobias Marschal
------

- Pangenome Graphs
- Pangenome-based genome inference
- A strategy for building and using a human reference pangenome
- Enabling multiscale variation analysis with genome graphs
- GraphAligner- rapid and versatile sequence-to-graph alignment
- MBG- Minimizer-based Sparse de Bruijn Graph Construction
- The design and construction of reference pangenome graphs with minigraph


Rayan Chikhi
------

- Minimizer-space de Bruijn graphs- Data structures based on k-mers for querying large collections of sequencing data sets- Data Structures to Represent a Set of k-long DNA Sequences




Adam M. Phillippy  (genome assembly mostly)
------

- HLA*PRG/LA – HLA typing from linearly projected graph alignments- Mash- fast genome and metagenome distance estimation using MinHash- Merqury- reference-free quality, completeness, and phasing assessment for genome assemblies
- HiCanu- accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads
- Purge Haplotigs- allelic contig reassignment for third-gen diploid genome assemblies- Identifying and removing haplotypic duplication in primary genome assemblies

